### DISCUSSION TITLE
Is there any tutorial for the boring stack?

### DISCUSSION THREAD

**Discussion by zuhrauin** [sailscastshq/boring-stack#75] [General]
A beginner's tutorial would surely help people dip their toes in the stack! I could imagine a portfolio/blog project as a start

**Comment by DominusKelvin** [comment]
Great idea @zuhrauin, I plan to do like a DHH style walkthrough of the stack and then follow it with a full blown course on Sailscasts.

I'll get around to doing this as early as possible in 2024.

**Reply by zuhrauin** [reply]
can't wait for the great news!

**Reply by DominusKelvin** [reply]
> can't wait for the great news!

Yeah sure I'll announce here too

**Reply by DominusKelvin** [reply]
Are you currently building anything with The Boring JavaScript Stack @zuhrauin ?

**Reply by zuhrasofyan** [reply]
_(Switch to my main github account now)_ 
Not right now. I am currently building things with Nextjs due to the necessity, but I always feel uncomfortable with all the issues of NextJS has. I used to build [the backend with sails and the frontend using Angularjs ](https://github.com/zuhrasofyan). Having a monorepo makes sense, and I always love the convention over configuration approach! So, going back to sails with this new stack will be a fresh breath of air for me. Would love to build something for my weekend side project using TBJS soon!

**Reply by DominusKelvin** [reply]
Oh that's awesome @zuhrasofyan I am currently writing the docs for TBJS and this is how simple it is to get started.
![CleanShot 2024-01-31 at 19 48 40@2x](https://github.com/sailscastshq/boring-stack/assets/24433274/5c20cb7e-5036-49d2-8125-908f311aa14e)
